addons/edi/models/edi_gateway.py

```diff
@@ -105,6 +105,11 @@ class EdiGateway(models.Model):
     can_initiate = fields.Boolean(string="Initiate Connections",
                                   compute='_compute_can_initiate')
     server = fields.Char(string="Server Address")
+    outgoing_server = fields.Char(string="Outgoing Server Address")
```

Comment:
Is this needed? we could use `server` instead

Reply:
Can do, assume we don't have a situation for inbound and outbound servers set